components/description-list/README.md

```diff
@@ -60,9 +67,9 @@ When the component width is less than the `breakpoint` property, the side-by-sid
 
 ## Description List Wrapper [d2l-dl-wrapper]
 
-The `d2l-dl-wrapper` can be combined with `descriptionListStyles` to apply styling and resize behavior to native `dl` elements.
+The `d2l-dl-wrapper` component can be combined with `descriptionListStyles` to apply styling and resize behavior to native `dl` elements.
 
-<!-- docs: demo live name:d2l-test-dl -->
+<!-- docs: demo live name:d2l-test-dl display:block -->
```

Comment:
Does the `display: block` do anything? I haven't seen this used before

Reply:
Yeah, this changes the `display` selector in the demo (defaults to flex). This felt like the cleanest way to resolve some of the resize issues I was seeing in the live demo.

Reply:
oo good to know, thanks!